standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -11322,6 +11322,13 @@ public NotificationEventResponse getNextNotification(NotificationEventRequest rq
         filterBuilder.setLength(filterBuilder.length() - 4); // remove the last " || "
         filterBuilder.append(") ");
       }
+      if (rqst.isSetEventTypeList()) {
+        for (String eventType : rqst.getEventTypeList()) {
+          parameterVals.add(eventType);
+          parameterBuilder.append(", java.lang.String para" + parameterVals.size());
+          filterBuilder.append(" && eventType == para" + parameterVals.size());
```

Comment:
this should be `||` instead of `&&`?

Reply:
You are right. Let me modify the test case to address the same.